src/big_query.ml

```diff
@@ -842,6 +842,7 @@ module Jobs = struct
 
 
   let add_gzip_headers ~use_gzip headers =
+    let _ = use_gzip in
```

Comment:
This is just to silence a warning because `use_gzip` is unused. I didn't want to just delete the aarg and edit the calling functions as I'd already made some other out-of-scope tweaks to silence `CCOpt` deprecation warnings

Reply:
This looks like an error from my previous PR - it should be:

```suggestion
    if use_gzip then
      Cohttp.Header.add_list headers gzip_headers
    else
      headers
```

Also, FYI to silence a warning like this you could also use the `~arg_name:alias` syntax:
```ocaml
let add_gzip_headers ~use_gzip:_ headers =
```